freva-rest/src/freva_rest/__init__.py

```diff
@@ -1,7 +1,7 @@
 import os
 from pathlib import Path
 
-__version__ = "2403.0.3"
+__version__ = "2408.0.0-dev2"
```

Comment:
there is no standard way to make it dynamic to inherits from the `freva-rest` version [here](https://github.com/FREVA-CLINT/freva-nextgen/blob/open-id-connect/freva-client/src/freva_client/__init__.py#L20) or vice versa? generally I'm not fan of defining version in `__init__.py` and instead of defining version in diff files, my suggestion is to define it in `pyproject.toml` and build the image based on the version in pyproject.

Reply:
I would like to be able to do this:

```python
import foo
print(foo.__version__)
```

This is important, because when people report issues the first we want to ask is what version they are using. 

I know what you could muck around with pkg_resources or importlib. But I think it's very clunky. Hence I will keep this __version__ constructs.